script.js

```diff
@@ -1,10 +1,13 @@
 function shifumi() {
     const user_choice = ['rock', 'paper', 'scissors']
-    const choice = prompt(`Choose one of the following : ${user_choice.join(', ')}.`)
-    if (user_choice.includes(choice)) {
-        alert(`Congratulations, you chose ${choice}`)
-    } else {
-        alert(`you did not choose one of the ${user_choice.length} choices`)
+    let choice = prompt(`Choose one of the following : ${user_choice.join(', ')}.`)
+    if (!user_choice.includes(choice)) {
+        alert(`Attention, ${choice} is not one of the ${user_choice.length} choices`);
+        return shifumi()
+        if (user_choice.includes(choice)) {
```

Comment:
useless check here. You don't need to use any condition.

More than that, this code will never be executed since you have a return before.

Reply:
one more change,
I delete the return line6, kept the second if and then return